web_monitoring/diffing_server.py

```diff
@@ -132,18 +144,27 @@ def caller(func, a, b, **query_params):
                                "".format(func.__name__, name))
     return func(**kwargs)
 
+class IndexHandler(tornado.web.RequestHandler):
+
+    @tornado.gen.coroutine
+    def get(self):
+        # Return a list of the differs.
+        # TODO Show swagger API instead.
+        self.write(repr(list(CONFIG)))
```

Comment:
I don’t think anything needs to change here, but this reminds me we should sit down and figure out whether this server should be publicly accessible (in which case we need to take time to work on simple security; e.g. the differ shouldn’t accept any old arbitrary URLs to diff). If not public, the utility of rendering Swagger docs here like we do for the DB API is pretty limited and documenting routes in a more readable text format, like Markdown, might be better.

Reply:
For v0, I'm pretty we should not worry about making it publicly accessible. I'd be happier with Markdown myself but willing to do swagger for consistency if we think it will eventually become public.

Reply:
Ok, let’s make sure we talk about this at the Wednesday meeting or sooner. Either way, I don’t think it needs to affect this PR.